src/v2/params/index.ts

```diff
@@ -54,6 +55,32 @@ function registerParam(param: Param) {
   declaredParams.push(param);
 }
 
+/**
+ * For testing.
+ * @internal
+ */
+export function clearParams() {
+  declaredParams.splice(0, declaredParams.length);
+}
+
+/**
+ * Declare a secret param, that will persist values only in Cloud Secret Manager.
```

Comment:
Nit: should start as third-person:

`Declares`

Reply:
Done.